apps/new-dealer/controllers/postcode.js

```diff
@@ -18,17 +18,17 @@ module.exports = class PostcodeController extends BaseController {
 
     if (_.startsWith(postcode, 'BT')) {
       req.sessionModel.unset('postcodeApiMeta');
-      req.sessionModel.unset('addresses');
+      req.sessionModel.unset(`${field}-addresses`);
       return callback();
     }
 
     const postcodesModel = new PostcodesModel();
```

Comment:
I did it for storage-postcode.js but forgot to do it for the this. Will do it now